Data Sources for each graph and table
================

This file details the sources of the data underlying all the figures and tables from the pA-QTL analysis.

-   **Figure 1:** pA annotations are from [polyA DB version 3](http://exon.umdnj.edu/polya_db/). Genomic sequences were extracted from the hg19 fasta file and the GTEx V7 variant annotation file. Both can be downloaded from [GTEx](https://gtexportal.org/home/datasets).

-   **Figure 2:** For box plots: [pAUI\_per\_sample.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/pAUI_per_sample.xlsx) provides both genotype and pAUI index information. p-values are in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx). UCSC coverage plots were produced using from [GTEx](https://gtexportal.org/home/datasets) BAM files.

-   **Figure 3:** **A.** Data is on [FastQTL\_PAS\_SNPs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_PAS_SNPs.xlsx). **B.** See [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx)

-   **Figure 4:** **A-B** For box plots see [pAUI\_per\_sample.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/pAUI_per_sample.xlsx), which provides both genotype and pAUI index information. p-values are in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx). UCSC coverage plots were produced using the BAM files from [GTEx](https://gtexportal.org/home/datasets) BAM files. **C-D** [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx).

-   **Figure 5:** **A-B** eQTL box plots: p-values and expression values are from from [GTEx](https://gtexportal.org/home/datasets). pA-QTL boxplots: see [pAUI\_per\_sample.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/pAUI_per_sample.xlsx), which provides both genotype and pAUI index information. p-values are in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx). **C-D** [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx).

-   **Figure 6:** For pA-QTL p-values see see [FastQTL\_PAS\_SNPs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_PAS_SNPs.xlsx). eQTL p-values, see [GTEx](https://gtexportal.org/home/datasets). For GWAS p-Values, see [**Table S4**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S4.xlsx) for references and links for downloading the summary statistics. LD are calculated from [GTEx](https://gtexportal.org/home/datasets) VCF file (eQTL plots), and [genome 1000 VCF files](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502) (GWAS plots). CLPP scores are in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx).

-   **Figure 7:** **A.** Distal pA usage indexes (PDUI) for 3'UTRs are from [APA atlas](https://hanlab.uth.edu/apa/). pA-QTL analysis output data is in [FastQTL\_3UTRs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_3UTRs.xlsx). **B.** pA annotations are from [polyA DB version 3](http://exon.umdnj.edu/polya_db/). Genomic sequences were extracted from the hg19 fasta file. Both can be downloaded from [GTEx](https://gtexportal.org/home/datasets). **C-D.** Data is on [**Table S5**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S5.xlsx).

-   **Figure S1:** For pA-QTL p-values see [FastQTL\_PAS\_SNPs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_PAS_SNPs.xlsx). LD is calculated from [GTEx](https://gtexportal.org/home/datasets) VCF file. Credible set information is in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx).

-   **Figure S2:** For pA-QTL p-values see For pA-QTL p-values see [FastQTL\_PAS\_SNPs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_PAS_SNPs.xlsx). GTEx p-values, see GTEx. LDs are calculated from the [GTEx](https://gtexportal.org/home/datasets) VCF file. CLPP scores are in Table S3.

-   **Figure S3:** For pA-QTL p-values see table S4. For GWAS p-Values, see [**Table S4**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S4.xlsx) for references and links for downloading the summary statistics. LDs are calculated from [genome 1000 VCF files](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502). CLPP scores are in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx).

-   **Figure S4:** For pA-QTL p-values see table S4. GTEx p-values, see GTEx. For GWAS p-Values, see [**Table S4**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S4.xlsx) for reference and links for downloading tht summary statistics. LD are calculated from [GTEx](https://gtexportal.org/home/datasets) VCF file (eQTL plots), and [genome 1000 VCF files](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502) (GWAS plots). CLPP scores are in [**Table S3**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S3.xlsx).

-   **Figure S5:** Distal pA usage indexes (PDUI) for 3'UTRs are from [APA atlas](https://hanlab.uth.edu/apa/). pA-QTL analysis output data is in [FastQTL\_3UTRs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_3UTRs.xlsx). B. pA annotations are from [polyA DB version 3](http://exon.umdnj.edu/polya_db/). See [**Table S5**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S5.xlsx) for GUGU variants.

-   **Figure S6:** See [MASH.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/MASH.xlsx) for lfsr values, posterior means and SDs. Caviar was run with posterior mean/ posterior SD, and LDs calculated from [GTEx](https://gtexportal.org/home/datasets) VCF file.

-   **Table S1:** pA annotations are from [polyA DB version 3](http://exon.umdnj.edu/polya_db/). Genomic sequences were extracted from the hg19 fasta file and the GTEx V7 variant annotation file. Both can be downloaded from [GTEx](https://gtexportal.org/home/datasets).

-   **Table S2:** Data according to [GTEx](https://gtexportal.org/home/datasets).

-   **Table S3:** For each tissue, FastQTL input was the pAUI indexes, [GTEx](https://gtexportal.org/home/datasets) VCF file and [covariats (from GTEx)](https://gtexportal.org/home/datasets) for the tissues. pAUI indexes per pA site per sample are on [pAUI\_per\_sample.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/pAUI_per_sample.xlsx). For credible set identification, caviar was run with t-scores input from [FastQTL\_PAS\_SNPs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_PAS_SNPs.xlsx) and LDs calculated from [GTEx](https://gtexportal.org/home/datasets) VCF file. eQTL data is from [GTEx](https://gtexportal.org/home/datasets). For GWAS columns, see [**TableS4**](https://github.com/ElkonLab/pA_QTLs/blob/master/Supplementary_Tables/Table_S4.xlsx) for references and links for downloading the summary statistics.

-   **Table S5:** A. Distal pA usage indexes (PDUI) for 3'UTRs are from (APA atlas)\[<https://hanlab.uth.edu/apa/>\]. pA-QTL analysis output data is in [FastQTL\_3UTRs.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/FastQTL_3UTRs.xlsx). For identification of GUGU variants, pA annotations are from [polyA DB version 3](http://exon.umdnj.edu/polya_db/). Genomic sequences were extracted from the hg19 fasta file and GTEx variants file. both can be downloaded from [GTEx](https://gtexportal.org/home/datasets).
