pA-QTLs
================

This repository contains the data underlying results and graphs of our pA-QTL analysis.

### Description of files content

-   [pAUI\_per\_sample.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/pAUI_per_sample.xlsx) The Sheet "Genotype" contains the genotype of all PAS SNPs that passed filtering criteria in at least one tissue, for all subjects (columns) that were included in our analysis (across all tissues). The values are coded according to the frequency of the PIA: 0 for PAS homozygote, 1 for heterozygote, 2 for PIA homozygote. Also, for each tissue, a sheet with the pAUI (cells) per PAS 3'UTR (rows) per subject (columns) for the tissue. Shown are 3'UTR whose PAS SNP passed filtering criteria.

-   [MASH.xlsx](https://github.com/ElkonLab/pA_QTLs/blob/master/MASH.xlsx) The output from [MASH](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6309609/) analysis. Rows are all the variants included in the analysis (variants that passed filtering criteria in all tissues), columns are tissues. lsfr sheet - local false sign rate, in PosteriorMean sheet - posterior mean, PosteriorSD posterior standard deviation.

### Authors

-   Eldad Shulman
-   Dr. Ran Elkon
